### PR TITLE
Use java 17 for Java ITs

### PR DESCRIPTION
**Pull request by christophe-zurn-sonarsource**
Fixes #

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -519,7 +519,7 @@ stages:
             -DskipTests=true
           publishJUnitResults: false
           javaHomeOption: 'JDKVersion'
-          jdkVersionOption: '1.11'
+          jdkVersionOption: '1.17'
```
Comment (martin-strecker-sonarsource) [inline azure-pipelines.yml:522]: We get

> Unhandled: Failed to find the specified JDK version. Please ensure the specified JDK version is installed on the agent and the environment variable 'JAVA_HOME_17_X64' exists and is set to the location of a corresponding JDK or use the [Java Tool Installer](https://go.microsoft.com/fwlink/?linkid=875287) task to install the desired JDK.

Shall we use the [JavaToolInstaller@0 task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/java-tool-installer-v0?view=azure-pipelines&viewFallbackFrom=azure-devops) or do we need to update the base image, @andrei-epure-sonarsource ?

**Review by martin-strecker-sonarsource — APPROVED** [review]
LGTM @costin-zaharia-sonarsource for a second look.

File: azure-pipelines.yml (hunk 1 above)
Comment (martin-strecker-sonarsource) [reply azure-pipelines.yml:522]: This was fixed by https://github.com/SonarSource/re-ci-images/pull/153

**Comment by costin-zaharia-sonarsource** [comment]
Pedantic: the description seems cut 
> Fixes #

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM! 

@christophe-zurn-sonarsource, I'll let you merge it to fix the description

**Comment by martin-strecker-sonarsource** [comment]
Thank you @christophe-zurn-sonarsource for taking care of this for us!

**Pull request merged**